src/validation/schema.js

```diff
@@ -38,6 +38,9 @@ const TYPE_SUGGESTIONS = [
   ],
 ]
 
+// As a convention, the type _SubgraphSchema_ is reserved to define imports on.
+const SUBGRAPH_SCHEMA = '_SubgraphSchema_'
```

Comment:
Not sure I like this name yet. The `Subgraph` prefix feels unnecessary. I'd almost want to make it something ridiculous like
```graphql
type _ @import(....) { ... }
```
One concern with using a `type` to add `@imports` to that I'm only realizing now is that the
```graphql
type _SubgraphSchema_ { ... }
```
definition needs to contain at least one field to be valid. We wouldn't want users to have to define anything inside the type though.

I now realize we can still make the
```graphql
# @import(...)
```
trick that I had initially planned work without messing with library internals. We could just take the schema file, extract the first comment and put `type _ .... { id: ID! }` around it. Then parse that the usual way and pull the directive information out of the resulting AST.

Reply:
What do you think? I'd strongly prefer that over using a special `_SubgraphSchema_` type. I think we could reuse most of the validation code here even.

Reply:
Where is the field required? Parsing works without the field and this type will be dropped in the schema merge.

Also, what's the draw of putting it in a comment? That is more jarring to me than a type (which we can rename so that it reads closer to its intent) with only imports on it.